File: core/src/main/java/com/example/anders/geoquiz/presenters/QuizView.java
package com.example.anders.geoquiz.presenters;

/**
 * Created by anders on 15/12/14.
 */
public interface QuizView {
    void saveCurrentQuestionIndex(int currentQuestionIndex);

    void saveUserCheated(boolean userCheated);

    void toastIncorrect();

    void toastCorrect();

    void setQuestionTextView(int id);

    void launchCheatActivity(boolean isTrue);

    void toastJudgement();
}

File: core/src/main/java/com/andersmurphy/geoquiz/presenters/QuizView.kt
package com.andersmurphy.geoquiz.presenters

/**
 * Created by anders on 15/12/14.
 */
interface QuizView {
    fun saveCurrentQuestionIndex(currentQuestionIndex: Int)
    fun saveUserCheated(userCheated: Boolean)
    fun toastIncorrect()
    fun toastCorrect()
    fun setQuestionTextView(id: Int)
    fun launchCheatActivity(isTrue: Boolean)
    fun toastJudgement()
}
